<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="CHAT_STACK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="CHAT_chatlist">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>441</width>
       <height>501</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="CHAT_BT_close">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>540</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>닫기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>441</width>
       <height>421</height>
      </rect>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="CHAT_BT_tsend">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>490</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>보내기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CHAT_LE_tsend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 </string>
     </property>
    </widget>
    <widget class="QTableWidget" name="CHAT_userlist">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>141</width>
       <height>321</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="CHAT_timeExit">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CHAT_gamecreate">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>390</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>방만들기</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="CHAT_gamerlist">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>141</width>
       <height>321</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>접속유저</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>게임방목록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CHAT_gameEnter">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>390</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>게임입장</string>
     </property>
    </widget>
    <widget class="QFrame" name="F_gamecreate">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>440</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="game_name">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>게임방이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>게임종류</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>인원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_agree">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>생성</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_cancel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
     <widget class="QComboBox" name="game_gamecb">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>게임선택</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>빙고</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>라이어게임</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="game_ppcb">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>인원선택</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2인</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4인</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6인</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8인</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="Game_userlist2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>181</width>
       <height>231</height>
      </rect>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="gr_sendtext">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>550</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="gr_sendbt">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>550</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>보내기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gr_roomclose">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
    <widget class="QListWidget" name="gr_chat">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>401</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="Game_userlist">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>181</width>
       <height>201</height>
      </rect>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>1</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>참여인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>대기실인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>제시어</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_initebt">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>초대</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_roomname">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>방이름</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_people">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_roomname_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>방이름</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_people_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_people_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>게임종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_gamename">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>인원</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_renewal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>새로고침</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="nameedit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connection">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>입장</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
